docs/development/codeguide.rst

```diff
@@ -17,13 +17,6 @@ Interface and Dependencies
   <https://github.com/astropy/astropy/blob/main/pyproject.toml>`_ file of the
   core package.
 
-* Usage of ``six`` and ``2to3`` is no longer acceptable.
-
-* `f-strings <https://docs.python.org/3/reference/lexical_analysis.html#f-strings>`_
-  should be used when possible, and if not, Python 3
-  formatting should be used (i.e. ``"{0:s}".format("spam")``)
-  instead of the ``%`` operator (``"%s" % "spam"``).
```

Comment:
This is enforced by [UP031 (printf-string-formatting)](https://docs.astral.sh/ruff/rules/printf-string-formatting/) and [UP032 (f-string)](https://docs.astral.sh/ruff/rules/f-string/#f-string-up032).